jsx/ato/ato_form.jsx

```diff
@@ -4,17 +4,26 @@ import { withFormik } from 'formik'
 
 const AtoForm = withFormik({
   displayName: 'AtoForm',
-  mapPropsToValues: props => ({
-    id: (props.data && props.data.id) || '',
-    name: (props.data && props.data.name) || '',
-    enable: (props.data && props.data.enable) || true,
-    control: (props.data && props.data.control) || false,
-    inlet: (props.data && props.data.inlet) || '',
-    period: (props.data && props.data.period) || 120,
-    pump: (props.data && props.data.pump) || '',
-    notify: (props.data && props.data.notify && props.data.notify.enable) || false,
-    maxAlert: (props.data && props.data.notify && props.data.notify.max) || 0
-  }),
+  mapPropsToValues: props => {
+    let data = props.data
+    if (data === undefined) {
+      data = {
+        enable: true,
+        notify: {}
+      }
+    }
+    return {
```

Comment:
Instead of doing this kind of checking would it be better to use a Class and call a Sanitize method in it? would be more readable in my opinion.

Reply:
This is specifically to handle a null input object.  I would prefer to use safe navigation operator instead of this, but it is not available in the version of babel we are using.  After 2.0 release, I would like to update babel to get access to the newer features.  Then I can replace this with something more like 
```
notify: props?.data?.notify?.enable ?? false,
```

Reply:
This is really weird for me because we are the master of the Data. So in this case we admit the data may be corrupted in a sense. After 2.0 it could be nice to check if we want to use strong types like we can in TypeScript for example